common/file_reader.h

```diff
@@ -410,6 +410,17 @@ class FileReader {
     itoa(v, value, 10);
     write_key_value(key, value);
   }
+  void write_key_value_float(const char* key, float f) {
```

Comment:
Add a comment that this always uses two decimals.

Reply:
comment added